python-package/lightgbm/basic.py

```diff
@@ -1710,13 +1710,15 @@ def eval_valid(self, feval=None):
         return [item for i in range_(1, self.__num_dataset)
                 for item in self.__inner_eval(self.name_valid_sets[i - 1], i, feval)]
 
-    def save_model(self, filename, num_iteration=-1):
+    def save_model(self, filename, start_iteration=0, num_iteration=-1):
```

Comment:
I understand that the current order of arguments is logically correct, but it'll break backward compatibility. Maybe it's better to swap the order of `start_iteration` and `num_iteration`?

Reply:
good point.